### ISSUE TITLE
Tolerate absence of flutter bin

### ISSUE THREAD

**Issue by klavs** [dint-dev/gpm#2]
When I switch computers and only have pub available, I want GPM to not to fail on flutter packages.

In https://github.com/gql-dart/gql we have some example Flutter packages, but I do not want to limit contributions from non-Flutter users. From users perspective GPM should skip Flutter packages. On CI we'd want to require Flutter so that an error is thrown.

Should Flutter be always required by default?

**Comment by terrier989** [comment]
Yep, I agree that gpm should skip Flutter packages if the developer doesn't have it. I'll implement this.
